### PR TITLE
Add sample internal_id when fetching sample files from hk

### PR DESCRIPTION
**Pull request by Vince-janv**
### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b fix-sample-fetching-from-hk -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by islean** [comment]
Does this add the sample_internal_id as tag? I think the bundle will always be the case bundle unless it is fastq delivery? (How the bundle is chosen. 
image
)

File: cg/meta/deliver/utils.py [hunk 1]
```diff
@@ -92,12 +94,13 @@ def should_include_file_case(file: File, sample_ids: set[str], workflow: str) ->
...
 
 
-def should_include_file_sample(file: File, workflow: str) -> bool:
+def should_include_file_sample(file: File, workflow: str, sample_internal_id: str) -> bool:
     """Check if file should be included in sample bundle.
 
     At least one tag should match between file and the sample tags.
     """
     file_tags = {tag.name for tag in file.tags}
+    file_tags.add(sample_internal_id)
```
Comment (islean) [inline cg/meta/deliver/utils.py:103]: Should we not exclude this tag for fastq deliveries? Just comparing with https://github.com/Clinical-Genomics/cg/blob/5872f73334059826529f056395831b9ca33070ff/cg/meta/deliver.py#L287-L305

**Comment by Vince-janv** [comment]
> Does this add the sample_internal_id as tag? I think the bundle will always be the case bundle unless it is fastq delivery? (How the bundle is chosen. image )
Good point, changed it to pass sample.internal_id through the functions

File: cg/meta/deliver/utils.py (hunk 1 above)
Comment (islean) [reply cg/meta/deliver/utils.py:103]: Oooo good thought! 😎

**Comment by Vince-janv** [comment]
Closing in favour of revert

**Pull request closed without merging**